connectors/cassandra/cassandra-api/src/main/java/org/teiid/cassandra/CassandraConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package org.teiid.cassandra;
```

Comment:
License header here also, every new file needs to have a license header

Reply:
got it, done